exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -318,7 +321,8 @@ func (prwe *PrwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	req.Header.Add("Content-Encoding", "snappy")
 	req.Header.Set("Content-Type", "application/x-protobuf")
 	req.Header.Set("X-Prometheus-Remote-Write-Version", "0.1.0")
-	req.Header.Set("User-Agent", "OpenTelemetry-Collector/"+version.Version)
+	FormatHeaderValues(req, prwe.startInfo.ExeName, prwe.startInfo.Version, prwe.startInfo.GitHash)
+	AddToUserAgent(req, "X-Prometheus-Remote-Write-Version/0.1.0")
```

Comment:
```suggestion
	FormatHeaderValues(req, "Prometheus-Remote-Write-Version", "0.1.0")
```
I would do this for consistency.  Also, the actual version should probably not be hard-coded since it's used in multiple places.  Adding a constant for it might be good.

Reply:
Yes sure, I will incorporate this change